pytorch_lightning/loops/epoch/training_epoch_loop.py

```diff
@@ -475,13 +475,15 @@ def _get_monitor_value(self, key: str) -> Any:
         # this is a separate method to aid in testing
         return self.trainer.callback_metrics.get(key)
 
+    def _is_check_val_epoch(self):
+        return (
+            self.trainer.enable_validation
+            and (self.trainer.current_epoch + 1) % self.trainer.check_val_every_n_epoch == 0
+        )
+
     def _should_check_val_fx(self, batch_idx: int, is_last_batch: bool) -> bool:
```

Comment:
We could rename this for consistency

```suggestion
    def _should_check_val_batch(self, batch_idx: int, is_last_batch: bool) -> bool:
```

Reply:
this checks for val_check_epoch too internally.